### PR TITLE
Do CI in stages

### PR DESCRIPTION
**Pull request by frej**

### CODE REVIEW

**Review by segeljakt — APPROVED** [review]
Cool, LGTM

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -23,10 +23,68 @@ jobs:
...
+      env:
+        A2M_BUILD: ${{ format('{0}/build', runner.temp) }}
+
+    - name: "Run arc-script test: check --examples"
+      run: ./.github/continuous-integration.sh cargo check --examples
+      env:
+        A2M_BUILD: ${{ format('{0}/build', runner.temp) }}
+
+    - name: "Run arc-script test: check --benches"
+      run: ./.github/continuous-integration.sh cargo fuzz run parse -- -runs=10000 -only-ascii
```
Comment (segeljakt) [inline .github/workflows/test.yml:80]: I think there is a bug here between `cargo check --benches` and `cargo fuzz`.

**Review by segeljakt — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (frej) [reply .github/workflows/test.yml]: Two things in the old script became one in this one, fixed.

**Review by segeljakt — APPROVED** [review]

**Comment by segeljakt** [comment]
LGTM

**Pull request merged**